RedHatAI/Llama-3.3-70B-Instruct-quantized.w4a16/accuracy/tasks.yml

```diff
@@ -0,0 +1,31 @@
+tasks:
```

Comment:
Manually run on ibm 02 and got following accuracy:

arc_challenge: acc_norm 0.6246
gsm8k: strict-match 0.8779
hellaswag: acc_norm 0.8368
mmlu: acc 0.8160
truthfulqa_mc2: acc 0.6034